usr/src/uts/common/sys/sysevent/eventdefs.h

```diff
@@ -203,6 +203,9 @@ extern "C" {
 #define	ESC_ZFS_CONFIG_SYNC		"ESC_ZFS_config_sync"
 #define	ESC_ZFS_SCRUB_START		"ESC_ZFS_scrub_start"
 #define	ESC_ZFS_SCRUB_FINISH		"ESC_ZFS_scrub_finish"
+#define	ESC_ZFS_SCRUB_ABORT		""ESC_ZFS_scrub_abort"
```

Comment:
looks like there's a typo here (two `""`)

Reply:
Blast it.  I thought I'd made that by patching.  Hold on a bit while I fix it.  grumble.